backend/dataall/modules/s3_datasets_shares/aws/s3_client.py

```diff
@@ -49,6 +49,9 @@ def create_bucket_access_point(self, bucket_name: str, access_point_name: str):
                 Bucket=bucket_name,
```

Comment:
nit: method name is a bit misleading now, is it a big change to rename to `try_create...` or `create_or_get...`?